scipy/_lib/_util.py

```diff
@@ -245,8 +245,195 @@ def float_factorial(n: int) -> float:
     return float(math.factorial(n)) if n < 171 else np.inf
 
 
+# SPEC 7
+def _transition_to_rng(old_name, *, position_num=None, end_version=None):
```

Comment:
I checked that this is copied almost veratim, but changes are intentional. Consider submitting them upstream.